docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
```

Comment:
maybe to much expected because codacy may have some weird expecations ( as seen in the "vorprojekt")